lib/p2p/Peer.ts

```diff
@@ -11,6 +11,9 @@ import { Packet, PacketDirection, PacketType } from './packets';
 import { HandshakeState, Address, NodeConnectionInfo } from '../types/p2p';
 import errors from './errors';
 import addressUtils from '../utils/addressUtils';
+import { compare } from 'semver';
+
+const minCompatibleVersion: string = require('../../package.json').minCompatibleVersion;
```

Comment:
this should be imported together with `version` in `Xud.ts`, and get passed in a similar way to `Pool`.